nexus/src/app/sagas/common_storage.rs

```diff
@@ -494,7 +494,19 @@ pub(super) async fn delete_crucible_snapshot(
     region_id: Uuid,
     snapshot_id: Uuid,
 ) -> Result<(), Error> {
-    // delete snapshot - this endpoint is synchronous, it is not only a request
+    // Unlike other Crucible agent endpoints, this one is synchronous in that it
+    // is not only a request to the Crucible agent: `zfs destroy` is performed
+    // right away. However this is still a request to illumos that may not take
+    // effect right away. Wait until the snapshot no longer appears in the list
+    // of region snapshots, meaning it was not returned from `zfs list`.
+
+    info!(
+        log,
+        "deleting region {} snapshot {}",
+        region_id.to_string(),
+        snapshot_id.to_string(),
```

Comment:
We can drop `.to_string()` or used named parameters here as well.

Reply:
done in 2fb0fb3a4